systemtest/apmservertest/command.go

```diff
@@ -199,7 +199,18 @@ func getRepoRoot() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	repoRoot = filepath.Clean(strings.TrimSpace(string(output)))
+	// When go workspaces is enabled then go list lists all the module paths in
```

Comment:
Would `go list -f '{{.Module.Dir}}' .` work?

Reply:
I didn't use this because, IIANM, this will only work if we run the command in the same directory as the `go.mod` file and I assumed that would not be desirable as in that case we could have used something like `os.Getwd`

Reply:
OK. Maybe we could use https://pkg.go.dev/runtime#Caller to locate the source code, and then the repo root relative to that. Not very robust to code changes/moving around, but I don't think that should happen often if at all.